cogs/ahk/ahk.py

```diff
@@ -295,24 +296,50 @@ async def cloudahk_call(self, ctx, code, lang='ahk'):
 		stdout, time = result['stdout'].strip(), result['time']
 
 		stdout = stdout.replace('`', '`\u200b')
+		file = None
+		encoded_out = stdout.encode('utf-8')
+		if len(stdout) < 1800 and stdout.count('\n') < 20 and stdout.count('\r') < 20:
+			#upload as plaintext
+			valid_response = 'No output.\n' if stdout == '' else '\n```autoit\n{0}\n```'.format(stdout)
+		elif len(encoded_out) < 100000 and AHK_GUILD_ID == ctx.guild.id:
+			#upload to ahkscript.org
+			data = {'name': 'Ace Bot (Clone)', 'code': stdout}
+			async with ctx.http.post('https://p.ahkscript.org/',data=data) as resp:
+				if resp.status != 200:
+					raise commands.CommandError('Pastebin failed.')
+				valid_response = ' No output.\n' if stdout == '' else f' Results too large. Pasted. {resp.url}\n'
+		elif len(encoded_out) < (1000*1000*4000): # should be 4mb
```

Comment:
`1000*1000*4000` can be put into a constant, and also missing spaces around *

Also max upload size for bots (and non-nitro users) is 8MB not 4MB